src/Service/SearchService.php

```diff
@@ -1,10 +1,12 @@
 <?php
 
-namespace Algolia\SearchBundle;
+namespace Algolia\SearchBundle\Service;
```

Comment:
What the reason behind moving this to this folder `Service`? If there a reason, it should be `Services` instead. As multiple services are inside.

I still would prefer to see this at rootLevel. what do you think?